core/src/main/java/org/springframework/security/authentication/UsernamePasswordAuthenticationToken.java

```diff
@@ -54,6 +54,10 @@ public UsernamePasswordAuthenticationToken(Object principal, Object credentials)
 		setAuthenticated(false);
 	}
 
+	public static UsernamePasswordAuthenticationToken unauthenticated(Object principal, Object credentials) {
```

Comment:
For consistency with other classes, please place both static methods after both constructors instead of immediately after their corresponding constructor.